tests/execute/reboot/out-of-session.sh

```diff
@@ -42,12 +42,12 @@ rlJournalStart
         tmt_reboot_command="export TMT_TEST_PIDFILE=/var/tmp/tmt-test.pid; export TMT_TEST_PIDFILE_LOCK=/var/tmp/tmt-test.pid.lock; export TMT_DEBUG=1; tmt-reboot"
 
         if [ "$PROVISION_HOW" = "container" ]; then
-            podman_exec="$(sed -nr 's/\s*Run command: (podman exec .*) \/bin\/bash.*cd.*/\1/p' tmt.output)"
+            podman_exec="$(sed -nr 's/\s*Run command: (podman exec .*) \/bin\/bash.*cd.*/\1/p' tmt.output | head -1)"
```

Comment:
Perhaps we can match for `cd ` instead of just `cd` (space after)?  
Tried to come up with something simpler, but failed:
`podman_exec=$(sed -n 's/.*Run command: \(podman exec [^ ]\+\).*/\1/p' tmt.output)`